TFT/src/User/API/SerialConnection.h

```diff
@@ -7,8 +7,8 @@ extern "C" {
 
 #include <stdbool.h>
 #include <stdint.h>
-#include "variants.h"  // for SERIAL_PORT_2 etc...
-#include "uart.h"      // for _UART_CNT etc...
+#include "variants.h"  // for SERIAL_PORT_2 etc.
```

Comment:
a very minor comment. In many languages etc is followed by ... (it was not an error to provide ... instead of .)

Reply:
I am no one to speak about other languages. At least in the English language, as far as I know, adding an ellipsis ('...') after 'etc' is wrong because it is an abbreviation. 'etc' should always end with a dot ('.'). Ellipsis is used when you don't want to complete a sentence or a list. 'etc.' itself is used to end a list.